src/utils/token-validity-check.ts

```diff
@@ -0,0 +1,10 @@
+export const checkTokenValidity = (token: string) => {
+  if (!token) {
+    return true;
```

Comment:
I think this function is quite confusing, you are returning true when the token is not valid. true = token is valid, false = token is invalid